lib/html/pipeline/dradis_escape_html_filter.rb

```diff
@@ -13,18 +13,24 @@ class Pipeline
     #
     # This filter does not write any additional information to the context hash.
     class DradisEscapeHTMLFilter < TextFilter
+      PROTOCOLS = %w{
```

Comment:
let's no shoot ourselves in the foot here, `http` and `https` are more than enough for our use case. I'd also hard-code them in the regex in L28